packages/core/database/encryption/encryption-schema-registry.js

```diff
@@ -16,9 +16,18 @@ const { logger } = require('./logger');
 const CORE_ENCRYPTION_SCHEMA = {
     Credential: {
         fields: [
+            // OAuth tokens
             'data.access_token',
             'data.refresh_token',
             'data.id_token',
+            // API key authentication (multiple naming conventions)
+            'data.api_key',
+            'data.apiKey',
+            'data.API_KEY_VALUE',
```

Comment:
can we not have data.api_key as standard? Having multiple makes later TS conversion more difficult and adds more code complexity for field validation.